docs/source/gbd2019_models/risk_exposures/stunting/index.rst

```diff
@@ -0,0 +1,337 @@
+
+**Chronic malnutrition**, also referred to as stunting ...
+
+.. note::
+  Include here a clinical background and overview of the risk exposure you're modeling. Note that this is only for the exposure; you will include information on the relative risk of the relevant outcomes, and the cause models for those outcomes, in a different document.
+
+.. _stunting1.1:
+
+Risk Exposures Description in GBD
++++++++++++++++++++++++++++++++++
+
+.. _stunting1.1.1:
+
+Case definition
+---------------
+
+Stunting, a sub-compoonent indicator of child growth failure (CGF), is based on a categorical definition using the WHO 2006 growth standards for children 0-59 months. Definitions are based on Z-cores from the growth standards, which were derived from an international reference population. Mild, moderate and severe categorical prevalences were estimated for each of the three indicators. Theoretical minimum risk exposure level (TMREL) for stunting was assigned to be greater than or equal to one standard deviation below the mean (-1 SD) of the WHO 2006 standard weight-for-height curve. This has not changed since GBD 2010.
+
++----------------------------------------------+
+| Stunting category definition (range -7 to +7)|
++=======+======================================+
+| TMREL |  >= -1                               |            
++-------+--------------------------------------+
+| Mild  |  < -1 to -2 Z score                  |
++-------+--------------------------------------+
+| Mod   |  < -2 to -3 Z score                  |
++-------+--------------------------------------+
+| Sev   |  < -3 Z score                        |
++-------+--------------------------------------+
+
+.. _stunting1.1.2:
+
+Input data
+----------
+
+Two types of input data are used in CGF estimation:  
+
+  1. **Tabulated report data**. This data does not report individual anthropometric measurements. It only reports the prevalence of forms of CGF in a sample size. For example, this data would may report a 15% prevalence of moderate stunting out of a nationally representative sample of 5,000 children.
+
+  2. **Microdata**. This data does have individual anthropometric measurements. From these datasources, GBD can see entire distributions of CGF, while also collapsing them down to point prevalences like moderate and severe CGF. 
+
+
+.. _stunting1.1.3:
+
+Exposure estimation
+------------------- 
+
+In modeling CGF, all data types go into ST-GPR modeling. We have STGPR models for moderate, severe, and mean stunting, wasting, and underweight. The output of these STGPR models is an estimate of moderate, severe, and mean stunting, wasting, and underweight for all under 5 age groups, all locations, both sexes, and all years. 
```

Comment:
I think in other documentation we've tried to either cite the quote, or use "we" to refer to us on simulation science---maybe edit for clarity?